src/views/Bitcoin.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { Grid, Text } from 'indigo-react';
+import { useWallet } from 'store/wallet';
+import { chunk } from 'lodash';
+
+import { useLocalRouter } from 'lib/LocalRouter';
+
+import ViewHeader from 'components/ViewHeader';
+import View from 'components/View';
+
+import CopiableAddressWrap from 'components/CopiableAddressWrap';
+import Highlighted from 'components/Highlighted';
+
+import { PsbtInput } from 'form/Inputs';
+import BridgeForm from 'form/BridgeForm';
+
+import * as bitcoin from 'bitcoinjs-lib';
+import { Buffer } from 'buffer';
+import { composeValidator, buildPsbtValidator } from 'form/validators';
+
+const hexToUx = hex => {
+  return (
+    '0x' +
+    chunk(
+      hex
+        .replace(/^0+/, '')
+        .split('')
+        .reverse()
+        .join(''),
+      4
+    )
+      .map(e => e.join(''))
+      .join('.')
+      .split('')
+      .reverse()
+      .join('')
+  );
+};
+
+export default function Bitcoin() {
+  const { pop } = useLocalRouter();
+  const { urbitWallet } = useWallet();
+
+  const [signedTx, setSignedTx] = useState();
+
+  const privKey = urbitWallet.unsafeGet().bitcoin.keys.private;
```

Comment:
If we don't have urbitWallet here, just explode. It's checked in `Point.js`, wanted to give these keys as props to my component but didn't figure out how to do it with this routing setup.